tests/test_fields.py

```diff
@@ -1186,74 +1186,88 @@ def test_minlength_maxlength(self):
         form = self.F()
         self.assertEqual(
             form.string1(),
-            """<input id="string1" minlength="1" name="string1" type="text" value="">""",
+            '<input id="string1" minlength="1"' ' name="string1" type="text" value="">',
```

Comment:
Remove the middle `' '` here.

Reply:
Oops… I added a break for every string literal and black rearranged this one to fit one line but didn't remove the `' '` in between.